docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -875,6 +887,76 @@ To process the received search command on the server side, follow these steps:
 > This feature supports Tizen 5.0 and Higher for Mobile and Wearable.
 
 
+## Updating and Retrieving Abilities
+To update the abilities on the server side:
+
+1. Create the media controller server handle using the `mc_server_create()`:
+
+   ```
+   ret = mc_server_create(&g_server_h);
+   ```
+   
+2. Set the abilities using the corresponding 'mc_server_set_XXX_ability()', or 'mc_server_set_ability_support()':
+   For example, to update the playback ability, set the ability for each playback actions using the 'mc_server_set_playback_ability()', and then update the abilities using the 'mc_server_update_playback_ability()':
```

Comment:
I updated. 
using 'mc_server_update_playback_ability()' is correct? or 
using the 'mc_server_update_playback_ability()' is correct?

Reply:
using 'mc_server_update_playback_ability()' is correct 

why I removed "the" is because there exists no qualifier after the callback name for example 

Just for your understanding: 

using the 'xxx_yyy_zzz()` callback. 

Here we have the word "callback" as the qualifier, thus we write "the" before the callback name.

Reply:
Done